AutomatedTesting/Gem/PythonTests/Prefab/tests/create_prefab/CreatePrefab_UnderAnEntity.py

```diff
@@ -13,7 +13,8 @@ def CreatePrefab_UnderAnEntity():
     Test is successful if the new instanced prefab of the child has the parent entity id
     """
 
-    CAR_PREFAB_FILE_NAME = 'car_prefab'
+    from pathlib import Path
+    CAR_PREFAB_FILE_NAME = Path(__file__).stem + 'car_prefab'
```

Comment:
`.stem` will remove the `.py`, so you will wind up with `CreatePrefab_UnserAnEntitycar_prefab`. Did you mean 
```suggestion
    CAR_PREFAB_FILE_NAME = Path(__file__).stem + '.car_prefab'
```
instead?

Reply:
Nope. I just need to prevent using a same name between batched EditorSharedTests. We don't need `.` here.